specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/blob.json

```diff
@@ -993,6 +993,14 @@
         "immutableStorageWithVersioning": {
           "$ref": "#/definitions/ImmutableStorageWithVersioning",
           "description": "The object level immutability property of the container. The property is immutable and can only be set to true at the container creation time. Existing containers must undergo a migration process."
+        },
+        "enableNfsV3RootSquash": {
+          "type": "boolean",
+          "description": "Enable NFSv3 root squash on blob container."
+        },
+        "enableNfsV3AllSquash": {
+          "type": "boolean",
+          "description": "Enable NFSv3 all squash on blob container."
```

Comment:
Can "enableNfsV3RootSquash" and "enableNfsV3AllSquash" both be enabled?  Is there any limitation between these 2 properties, and between the storage account property "EnableNfsV3"? 
Ask this since in File share, can only choose one from RootSquash/AllSquash.